tools/dune

```diff
@@ -48,3 +48,10 @@
   )
   (section bin)
   (package ocaml))
+
+(executable
+  (name merge_archives)
+  (modes native)
+  (flags -nostdlib)
```

Comment:
(Why is this needed? the code clearly shows that the stdlib is actually used,
presumably transitively from the other dependencies.)

Reply:
I was following the stanzas for the other executables, but in fact it seems not to be needed.  Removed.